CHANGELOG.md

```diff
@@ -5,6 +5,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.3.0] - 2020-06-29
+
+### Changed
+
+- Add Javascript linter.
+- Trigger the markdown link checker on commit on markdown files and linter only.
+
+### Fixed
+
+- Upgrade GitHub Actions virtual enviroment to Ubuntu LTS 20.04.
+TODO rebase after previous, upgrade version in package.json
```

Comment:
think maybe there's work left here to remove a TODO in a CHANGELOG?

Reply:
Sure, this was a comment for myself.